tests/ballerina-test/src/test/java/org/ballerinalang/test/mime/MimeTest.java

```diff
@@ -272,7 +272,8 @@ public void testLargePayload() {
             HTTPCarbonMessage response = Services.invokeNew(serviceResult, cMsg);
             Assert.assertNotNull(response, "Response message not found");
             String temporaryFilePath = ResponseReader.getReturnValue(response);
-            boolean isCorrectTempFileCreated = temporaryFilePath.startsWith("/tmp/ballerinaBinaryPayload");
+            Assert.assertNotNull(temporaryFilePath);
+            boolean isCorrectTempFileCreated = !temporaryFilePath.isEmpty();
```

Comment:
cant we do something like below here,
`Assert.assertFalse( "======== error message =====", temporaryFilePath.isEmpty());`